android/app/src/main/AndroidManifest.xml

```diff
@@ -40,7 +40,7 @@
     </service>
     <!-- END Required by react-native-push-notification -->
 
-    <activity android:name=".MainActivity" android:launchMode="singleTask" android:label="@string/app_name" android:configChanges="keyboard|keyboardHidden|orientation|screenSize" android:windowSoftInputMode="adjustPan">
+    <activity android:name=".MainActivity" android:launchMode="singleTask" android:label="@string/app_name" android:configChanges="keyboard|keyboardHidden|orientation|screenSize" android:windowSoftInputMode="adjustResize">
```

Comment:
Did you check if this change can impact any forms inside the app?

For example check the services search


### without fix (master version)
<img src="https://user-images.githubusercontent.com/822471/65761662-89952700-e11f-11e9-933f-54ff223b208c.png" height="600"/>

### with fix
<img src="https://user-images.githubusercontent.com/822471/65761673-94e85280-e11f-11e9-886b-28fb959c5ccd.png" height="600"/>

Reply:
I forgot this specific case, I tried the form to insert a credit card and the form to insert the manual payment data. I will do other checks!